core-java-modules/core-java-collections-5/src/main/java/com/baeldung/immutables/ImmutableCollectionsExample.java

```diff
@@ -0,0 +1,35 @@
+package com.baeldung.immutables;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+public class ImmutableCollectionsExample {
```

Comment:
You can skip the whole class and create the objects directly into the tests.

Reply:
Ok. I m going to do this now. I m deleting this class.